html/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="nav">
+        <div class="container">
+            <div class="logo">
+                Hello Quizzer
+            </div>
+            <div class="breadcrumbs">
+                <a href="#home" class="active">Navbar</a> 
+            </div>
+            
+            <a href="#about">About</a>
+            <a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776;</a>
+        </div>
+    </div>
+    <div class="container">
+        <h1>Hello Quizzer</h1>
+
+        <section id="questions">
+            <span>What is the Capital City of Uganda?</span>
+        </section>
+        <section id="answer">
```

Comment:
Could you also look into when to use section tags?(they usually have headings) http://html5doctor.com/element-index/#section